components/compliance-service/integration_test/stats_server_read_failures_test.go

```diff
@@ -3,7 +3,7 @@ package integration_test
 import (
 	"testing"
 
-	iam_v2 "github.com/chef/automate/api/interservice/authz/v2"
+	iamV2 "github.com/chef/automate/api/interservice/authz/v2"
```

Comment:
why? 
https://blog.golang.org/package-names

Reply:
I'm a little bit torn here. I know that both snake and camel case are discouraged in package names.  
snake_case is more generally discouraged in go.
in a way, I think that just naming it iam is the way to go here.  what do you think?

Reply:
I find myself needing to use multi-word aliases for packages from time to time.  snake_case and camelCase being antipatterns make this difficult.

Reply:
ya, in the go examples they do "syscall" for system call. but it can get messy looking. our package naming (and aliasing) is really all over the place in this codebase tbh